ports/windthe/gameinfo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./Wind, The.sh</path>
```

Comment:
Is there  a reason why It's called Wind, The? Some CFws use the .sh and Name as Sorting so it will be under W while it's The Wind and not just Wind. If I look for The Wind i look for T not W.
In #Crew-Poll open a principle decision poll how to deal with it.

Reply:
Had this convention drummed into me throughout high-school :)

However, I checked the repo to find several ports folders beginning with "the," so I won't deviate from our convention.

Reply:
> Had this convention drummed into me throughout high-school :)
> 
> However, I checked the repo to find several ports folders beginning with "the," so I won't deviate from our convention.

I don't think we ever talked about it :D